sdk-lib/src/main/java/cash/z/ecc/android/sdk/jni/RustBackend.kt

```diff
@@ -25,14 +26,14 @@ internal class RustBackend private constructor(
     override val network: ZcashNetwork,
     val birthdayHeight: BlockHeight,
     val dataDbFile: File,
-    val cacheDbFile: File,
+    val fsBlockDbRoot: File,
     override val saplingParamDir: File
 ) : RustBackendWelding {
 
     suspend fun clear(clearCacheDb: Boolean = true, clearDataDb: Boolean = true) {
         if (clearCacheDb) {
-            twig("Deleting the cache database!")
-            cacheDbFile.deleteSuspend()
+            twig("Deleting the cache database and CompactBlock files!")
+            fsBlockDbRoot.deleteSuspend()
```

Comment:
This currently is for deleting a single file, and I think it will fail.  I think we need a recursive delete.